code-challenges.js

```diff
@@ -13,7 +26,12 @@ var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+rearrangeBaby = (array) => {
+    return array.sort((a,b) => a-b).filter(value => typeof value === "number" &&value%2===1 || value%2===-1)
+}
 
+console.log(rearrangeBaby(fullArr1))
```

Comment:
your return for this array is `[ 9, 7, 199, -9 ]` so not quiiiite right. but close